framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/DeploymentScenarioBuilderFactory.java

```diff
@@ -52,5 +53,7 @@
     WorkbenchMonitoringSettingsBuilder getWorkbenchMonitoringSettingsBuilder();
     SmartRouterSettingsBuilder getSmartRouterSettingsBuilder();
 
+    LdapSettings getLdapSettings();
```

Comment:
I think we should have a builder here, to separate creating of LdapSettings from the object itself.